1.Exploratory_Data_Analysis/README.md

```diff
@@ -6,4 +6,13 @@ Where 'waves' of data are data from the same plate but using different channels
 is for clustering of cells across all features. The goal is to aim to see if treatments cluster or not.
 ```cell_count_analysis.ipynb``` is for qc analysis to assess if there are cell count drop offs for a given treatment.
 
+```umap_analysis.ipynb``` runs umap analysis on all image features across all treatments to see if replicates seperate or not.
+
+```2.extended_umap_analysis.ipynb``` runs umap analysis on all image features across some treatments but creates one umap per image channel to decipher which channels are the most variable across single cells.
```

Comment:
Recommend adding more info on what "some treatments", it would be important to describe why you are looking at these specific treatments.